### ISSUE TITLE
Gigabit QA: Deselect tab on Terms and Conditions page (and other nested pages)

### ISSUE THREAD

**Issue by flukeout** [mozilla/learning.mozilla.org#2320]
Recently we came up with a convention that if you're in a nested page in a tab, we would deselect that tab. This gives you a more obvious affordance to get back to the parent tab.

I'd recommend the same for the Terms and Conditions page...

![image](https://cloud.githubusercontent.com/assets/25212/19203765/f818bfc8-8c8d-11e6-9a4a-6e40040ebc78.png)

#2228

**Comment by gvn** [comment]
That seems like an odd convention to me. 

Wouldn't it be better to have a breadcrumb or something? IA-wise you are in the Apply section.

**Comment by flukeout** [comment]
Yeah, I'd be into seeing what a breadcrumb could look like. Feels weird without some indicator that you're not on the main Apply page.
